lib/jekyll/tags/include.rb

```diff
@@ -2,14 +2,6 @@
 
 module Jekyll
   module Tags
-    class IncludeTagError < StandardError
-      attr_accessor :path
-
-      def initialize(msg, path)
-        super(msg)
-        @path = path
-      end
-    end
 
```

Comment:
Some plugins use this class: https://github.com/search?l=Ruby&q=IncludeTagError+language%3ARuby&ref=advsearch&type=Code&utf8=%E2%9C%93